app/controllers/pensions/lifetimeAllowance/LifeTimeAllowanceAnotherWayController.scala

```diff
@@ -74,7 +75,7 @@ class LifeTimeAllowanceAnotherWayController @Inject()(implicit val cc: MessagesC
               pensionsCYAModel.copy(
                 pensionLifetimeAllowances = viewModel.copy(
                   pensionPaidAnotherWayQuestion = Some(yesNo),
-                  pensionPaidAnotherWay = if (yesNo) viewModel.pensionPaidAnotherWay else None)
+                  pensionPaidAnotherWay = if (yesNo) charges.LifetimeAllowance(viewModel.pensionPaidAnotherWay.amount, viewModel.pensionPaidAnotherWay.taxPaid) else LifetimeAllowance(None, None))
```

Comment:
Switch to merely LifetimeAllowance(), once both values default to 'None'.

Reply:
Change implemented